docs_outdoornav_user_manual/web_user_interface/ui_overview.mdx

```diff
@@ -421,3 +412,203 @@ below.
+
+    See the [Custom Tasks](../features/custom_tasks.mdx) section 
+    for details on how to develop custom tasks for your application.
+
+
+:::note
+
+If a Waypoint/Goalpoint has more than one task assigned to it, the icon will
+be replaced with a sheet of paper icon like so: <img src="/img/outdoornav_images/task_multi.png" width="30px"/>
+
+:::
+
+### Autonomous Docking
+
+<center>
+  <figure>
+    <img src="/img/outdoornav_images/ui_map_with_dock.png" />
+    <figcaption>Dock Icon</figcaption>
+  </figure>
+</center>
+
+### Docking The UGV
+
+To dock the UGV autonomously the user can use the following methods:
+
+#### Dock (Local)
+
+If there is a valid dock within it's drivable range the UGV will attempt to dock. 
+This docking method can be done by selecting the Dock button in the bottom bar or by adding a 
+"Dock Robot (Local)" task to either an event or a mission point. 
+
+#### Dock (Radius)
+
+If the UGV is within the specified docks radius the UGV will first navigate to a predock location 
+and then attempt to dock. When in Waypoint Mode a user can dock via this method as follows:
+
+-   Enable the "Edit Mission" toggle.
+-   Maneuver the UGV so that is somewhere within the dock's radius.
+-   Click the dock that the UGV will be docking at and select "Dock Robot Here".
+
+This docking method can also be accomplished by adding a "Dock Robot (Radius)" task to either 
+an event or a mission point. A dock will need to be provided for the task to function which can be done 
+through the task settings modal.
+
+#### Dock (Network) 
+
+:::note 
+
+This mode is only available in [Map mode](ui_map_mode.mdx). Please see the Map mode page for further
+details on map generation.
+
+:::
+
+If the UGV and the dock are within a map's driveable aread the UGV will first navigate to a 
+predock location based on the map paths available. A user can dock manually using this method as follows:
+
+-   Enable the "Edit Map" toggle. 
+-   Maneuver the UGV so that is within the driveable range of the map. 
+-   Ensure the dock is within the driveable range of the map.
+-   Right click the dock that the UGV will be docking at and select "Dock Robot Here".
+
+This docking method can also be accomplished by adding a "Dock Robot (Network)" task to either an 
+event or a mission point. A dock will need to be provided for the task to function which can be done 
+through the task settings modal.
+
+### Undocking The UGV
+
+To undock the UGV autonomously, the user can apply a "Undock Robot" task to a mission event or 
+point. The user can also click the Undock button in the bottom bar's left hand corner.
+
+### Compatibility with a Doghouse
+
+In order for the autonomous docking feature to be compatible with a doghouse, the
+doghouse must conform to a few specifications:
+
+-   Must be installed on a flat surface, and have no elevation change between the
+    charge-deck and the outdoor surface (ie. no ramp into the doghouse).

```

Comment:
deck -> dock?

Reply:
I've changed it to charge-dock but @jmastrangelo-cpr perhaps this is wrong?